### DISCUSSION TITLE
More push notifications

### DISCUSSION THREAD

**Discussion by andersonalex** [community/community#3206] [Mobile] [answered]
I would love to be able to receive push notifications for anything I could get an email for (PR comments especially)

Thanks!!

**Comment by candyho** [comment, marked as answer]
Hey thanks for your feedback! Supporting more push notification types is definitely something on our radar. Hopefully we'll have some updates to share with you soon. Stay tuned!

**Reply by LiroyvH** [reply]
There any news on this? Notifications for issues you're subscribed to, most notably the ones you created, is really necessary to have haha.

**Reply by johnbeadle** [reply]
Can I ask if there is a reason I only get push notifications on my iPad and not my iPhone for the mobile app?

My iPad is often not with me but is swallowing all my push notifications and none make it through to my iPhone 😬

**Reply by GitSparTV** [reply]
@candyho A year have been passed, what's the point of email junk? I watch a repository -- I want notifications.
What's the priority on it?

**Reply by candyho** [reply]
Hey, we have shipped a couple of new push notification types last year, including [Assigned, Review Requested, Deployment Review and Pull Request Review](https://github.blog/2021-03-30-new-push-notifications-scheduling-releases-github-mobile/). Are there any specific push notifications that you'd like us to add and why's that? Are you wanting to get **all** the notifications related to an issue that you have created? Let us know!

**Reply by GitSparTV** [reply]
@candyho Hi. I'm watching several repos for their commits and their issues and I want to stay tuned on them. But these types of notifications can be received only by email.
In the app I have all notifications enabled for the types you mentioned, it's great, I don't have large projects so I'm fine to receive everything immediately. I follow issues where I am not even mentioned.

**Reply by GitSparTV** [reply]
In short: all things you can subscribe to I want to receive through in-app notifications

**Reply by okaestne** [reply]
I also would like to receive push notifications for all things that can be found in the Notifications tab. Subscribed issues and repo activity such as new PRs or releases.

**Reply by GitSparTV** [reply]
Wow it has been 10 month ago when I wrote here last time, could be more

**Reply by okaestne** [reply]
> Wow it has been 10 month ago when I wrote here last time, could be more

Just got notified about your answer, via mail :)
